packages/contracts/contracts/voting/admin-address/AdminAddress.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
+
+import {PluginCloneable} from "../../core/plugin/PluginCloneable.sol";
+import {IDAO} from "../../core/IDAO.sol";
+
+/// @title AdminAddress
+/// @author Aragon Association - 2022.
+/// @notice The Admin address governance plugin, to serve as a DAO's primary full controller.
```

Comment:
> primary full controller
It doesn't have a primary status. Other governance plugins could be installed as well (which doesn't necessarily make sense).

```suggestion
/// @notice The admin address governance plugin giving execution permission on the DAO to a single address
```